draft-trammell-taps-post-sockets.md

```diff
@@ -807,7 +819,7 @@ Local and Remote, depending on how it was configured.
 // open a connection to a server using an existing Association and send some data,
 // which will be sent early if possible.
 func sayHelloWithAssociation(association Association) {
-    carrier := InitiateWithAssociation(association)
+    carrier := association.Initiate()
```

Comment:
this should probably not be called initiate(); maybe bind() or getCarrier() or createCarrier() or initiateCarrier()

Reply:
Ah in the new section below it's actually called GetCarrier()

Reply:
yep will fix

Reply:
since GetCarrier() is not what's meant here, .Initiate() is actually correct, so no change necessary.

Reply:
What's initiate supposed to be? InitiateCarrier() at least? Or CreateCarrier() I would say

Reply:
This is called .Initiate() since it does the same thing as Initiate() (bare, not as an Association method).

Reply:
Yup, I like .Initiate() here.